interpreters/src/tests.rs

```diff
@@ -334,7 +336,7 @@ where
 
 #[tokio::test]
 async fn test_interpreters_rocks() {
-    common_util::tests::init_log_for_test();
+    // common_util::tests::init_log_for_test();
```

Comment:
Why make it a comment?

Reply:
Oops, because it did not work in debug mode, i will revert it;